wpilibOldCommands/src/test/java/edu/wpi/first/wpilibj/command/CommandSupersedeTest.java

```diff
@@ -60,12 +60,8 @@ void oneCommandSupersedingAnotherBecauseOfDependenciesTest() {
   void commandFailingSupersedingBecauseFirstCanNotBeInterruptedTest() {
     final ASubsystem subsystem = new ASubsystem();
 
-    final MockCommand command1 =
-        new MockCommand(subsystem) {
-          {
-            setInterruptible(false);
-          }
-        };
+    final MockCommand command1 = new MockCommand(subsystem);
+    command1.setInterruptible(false);
```

Comment:
There was a warning discouraging double-braced init. The equivalent is cleaner anyway, imo.